build.sh

```diff
@@ -63,6 +72,30 @@ do
 	u)
 		UPLOAD="YES"
 		;;
+	r)
+		case $OPTARG in
+			13.2)
```

Comment:
move parsing the release version out of options parsing underneath shift "$((OPTIND-1))"